Add quick fix documentation and update the screen shot for start with config (#581)
OpenLiberty/liberty-tools-intellij: docs/user-guide.md

### Start your application in dev mode with configuration

> To start your application in dev mode with custom configuration, select the **Start...** action for your application in the Liberty tool window. The action opens a Liberty Edit Configuration dialog. You can specify parameters for the [Liberty Maven dev goal](https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#additional-parameters) or [Liberty Gradle dev task](https://github.com/OpenLiberty/ci.gradle/blob/main/docs/libertyDev.md#command-line-parameters). You can also check the box to start your application in dev mode in a container.

Reviewer: LGTM except I think we've been using bold rather than quotes for UI text:

**Run in container** checkbox

> To start your application in dev mode with custom configuration, select the **Start...** action for your application in the Liberty tool window. The action opens a Liberty Edit Configuration dialog. You can specify parameters for the [Liberty Maven dev goal](https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#additional-parameters) or [Liberty Gradle dev task](https://github.com/OpenLiberty/ci.gradle/blob/main/docs/libertyDev.md#command-line-parameters). You can also check the box to start your application in dev mode in a container.

Author: 👍

> To start your application in dev mode with custom configuration, select the **Start...** action for your application in the Liberty tool window. The action opens a Liberty Edit Configuration dialog. You can specify parameters for the [Liberty Maven dev goal](https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#additional-parameters) or [Liberty Gradle dev task](https://github.com/OpenLiberty/ci.gradle/blob/main/docs/libertyDev.md#command-line-parameters). You can also check the box to start your application in dev mode in a container.

Author: Fixed the bold.

![Eclipse LSP4MP microprofile-config.properties completion](images/LSP4MP-mp-properties-completion.png)

> 3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a supported Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a quick-fix (code action) appears.

Reviewer 2: ```suggestion
3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a quick-fix suggestion will appear. To see more quick-fix suggestions, click on the "More actions..." link.
```

Reviewer: same here:

click on the **More actions...** link.

Author: 👍

Author: Done.

> 3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a Jakarta EE or MicroProfile diagnostic. A pop-up dialog will appear which contains the diagnostic message and a quick-fix suggestion, if available. To see more quick-fix suggestions, click on the **More actions...** link.

Reviewer: ```suggestion
3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a Jakarta EE or MicroProfile diagnostic. A pop-up dialog appears that contains the diagnostic message and a quick-fix suggestion, if available. To see more quick-fix suggestions, click on the **More actions...** link.
```

Author: Looks good, thank you.
